### ISSUE TITLE
REST API /engine/start v1.9.0 breaks when starting a started tenant. 

### ISSUE THREAD

**Issue by nmarus** [sitewhere/sitewhere#102]
This only happens when sending the REST API call to start a tenant that is already started, I get the following response:

```json
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":1,"message":"Preparing tenant"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":2,"message":"Starting tenant Groovy script engine"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":3,"message":"Starting asset management cache provider"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":4,"message":"Starting asset management"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":5,"message":"Starting device management cache provider"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":6,"message":"Starting device management"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":7,"message":"Starting device event management"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":8,"message":"Starting schedule management"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":9,"message":"Starting asset module manager"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":10,"message":"Starting search provider manager"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":11,"message":"Starting event processing subsystem"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":12,"message":"Starting device communication subsystem"}
{"taskName":"START TENANT 'Test1'","totalOperations":14,"currentOperation":13,"message":"Starting schedule manager"}
{"result":"Failed","message":"java.lang.RuntimeException: Unable to create Quartz scheduler for schedule manager."}
```

However, the tenant goes into a state of 'error' and I can no longer start this or ANY new tenants without rebooting/restarting sitewhere.

Everything works fine if the tenant is stopped when attempting to start it. 

I am working around this by first checking if a tenant is started before sending a start, but wanted to let you know that this happens under these circumstances.

**Comment by derekadams** [comment]
Thanks for reporting the issue. We are about to release 1.10.0 and there is a chance it's fixed in there because we refactored some of the tenant lifecycle code. If not, we will schedule a fix for the 1.11.0 release. Sorry for the inconvenience!

**Comment by nmarus** [comment]
No worries. Easy workaround. Excited to play with 1.10 when you guys release. Loving this project. Great work!
